coreapi/models.py

```diff
@@ -0,0 +1,85 @@
+''' creates a db handler class to do crud operations on the API '''
+
+
+class Question():
+    ''' Question class '''
+    idCounter = 0
+
+    def __init__(self, title=None, body=None):
+        ''' intiate default values for the class '''
+        self.title = title
+        self.body = body
+        Question.idCounter += 1
+
+    def serialize(self):
+        ''' return json representation of the class '''
+        question = {
+            '_id': Question.idCounter,
+            'question_title': self.title,
+            'question_body': self.body
+        }
+        return question
+
+
+class Answer():
+    ''' Answer class '''
+
+    def __init__(self, answers=[]):
```

Comment:
@bozicschucky Mutable objects such as lists should never be used as function arguments. Also I don't see why you initialize the answer class with a list of answers instead of asnwer attributes.